about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/aboutstyles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+    <title>About Women Tech</title>
+</head>
+<body>
+    <header class="navigation">
+        <nav class="nav-list">
+          <div class="header">
+            <ul>
+                <li><i class='fa fa-facebook-official'></i></li>
+                <li><i class='fa fa-twitter'></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">Mypage</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>

```

Comment:
Let's exclude this part from the navbar. You can assign a separate div for this part since at this point it doesn't serve any purpose and is there only for presentational purposes.  Do the same for the index.html

Reply:
Okay, thank you for reviewing my project. I'll correct it.